src/components/modal-base/ModalTrigger.js

```diff
@@ -2,25 +2,28 @@ import React, { Component, Fragment, cloneElement } from 'react';
 import PropTypes from 'prop-types';
 
 class ModalTrigger extends Component {
-    defaultEventProps = {
-        onClick: this.handleOpen,
-    };
+    defaultEventProps = null;
+    state = { open: false };
+
+    constructor() {
+        super();
 
-    state = { isOpen: false };
+        this.defaultEventProps = {
```

Comment:
The constructor is not needed here, the `defaultEventProps` can be defined directly in the class